core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -76,6 +79,20 @@ angular.module('oppia').directive('storyNodeEditor', [
               }
             }
           };
+          var _isExpIdValid = function(expId) {
+            // If user has not yet entered an exploration id
+            if (expId === null) {
+              return true;
+            }
+            ExplorationSummaryBackendApiService.loadPublicExplorationSummaries(
+              [expId]).then(function(summaries) {
+              if (summaries.length !== 1) {
+                return false;
```

Comment:
instead of if-else can we make it linear? `return summaries.length === 1`

Reply:
Done